Dockerfile-server

```diff
@@ -57,12 +57,14 @@ ENV GUNICORN_WORKERS=1
 ENV GUNICORN_THREADS=1
 ENV GUNICORN_BIND="0.0.0.0:8000"
 ENV GUNICORN_TIMEOUT=400
+ENV GUNICORN_MAXREQUESTS=2000
```

Comment:
How did you end up with 2000?

Reply:
Ballparking! Feel very very free to suggest somthing else. Im testing around a bit on stage, but as we know we can't quite replicate, so...

Reply:
Honestly, I feel we are giving mem back quite a bit, its not just explosive linear growth, though it is certainly growth over time.

Reply:
I don't have any better number in mind, so 2000 is fine by me, let's hope it works!